vernac/vernacstate.mli

```diff
@@ -67,10 +83,10 @@ module Proof_global : sig
 
   val get_all_proof_names : unit -> Names.Id.t list
 
-  val copy_terminators : src:Lemmas.Stack.t option -> tgt:Lemmas.Stack.t option -> Lemmas.Stack.t option
+  val copy_terminators : src:Stack.t option -> tgt:Stack.t option -> Stack.t option
 
   (* Handling of the imperative state *)
-  type t = Lemmas.Stack.t
+  type t = Stack.t
```

Comment:
What's the point of this alias?

Reply:
Was to reduce churn in cut & paste.

Reply:
I tried removing it and it seems it's only used in this file, would you add that to the PR?